manim/scene/scene.py

```diff
@@ -146,6 +146,9 @@ def __init__(
             random.seed(self.random_seed)
             np.random.seed(self.random_seed)
 
+    def clear():
+        self.remove(*self.mobjects)
```

Comment:
Is it actually necessary to call `Scene.remove`, or would it be sufficient to simply set `self.mobjects = []`? The latter would be more performant, perhaps it is worth a try.

```suggestion
        self.remove(*self.mobjects)
```